.github/workflows/connector-performance-command.yml

```diff
@@ -179,6 +179,8 @@ jobs:
         run: |
           export PATH="$PATH:/root/.local/bin"
           ci_credentials connectors-performance/$HARNESS_TYPE write-to-storage
+          connector_name=$(echo ${{ inputs.connector }} | cut -d / -f 2)
```

Comment:
IIRC the `connectors/` prefix is optional so perhaps instead of piping into `cut`, consider removing the prefix using `sed` like this: 
```
connector_name=$(echo ${{ inputs.connector }} | sed 's,.*/,,')
```

FYI the inputs.connector is required to match `^((connectors|bases)/)?[a-zA-Z0-9-_]+$"`. Is `bases` still relevant? Consider removing support for it.

Reply:
the input from action would now be a dropdown so "connectors" would be guaranteed in that case. base is not required.

If triggering from command then this makes sense!

<img width="340" alt="Screenshot 2023-11-14 at 9 52 34 AM" src="https://github.com/airbytehq/airbyte/assets/5505742/94f4c1d8-aad9-40d6-9c4d-8ff36d347d20">